azure_pipelines/spm-framework.yml

```diff
@@ -0,0 +1,311 @@
+variables:
+  - name: 'releaseVersion'
+    value: null
+  - name: 'commonCoreReleaseVersion'
+    value: null
+  - name: 'frameworkChecksum'
+    value: null
+  - name: 'repositoryName' # Name of the repository
+    value: 'AzureAD/microsoft-authentication-library-for-objc'
+  - name: 'repositoryBranch' # Name of the branch where version number will be updated for files that contain version number.
+    value: 'ameyapat/spm-framework-build'
```

Comment:
I assume this is for testing purposes

Reply:
Yes, just for testing. Will replace with 'master' once all testing is done.